### PR TITLE
Added sprite for empty syringe gun, loading sound

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Normal syringe guns will now show when they are empty. They also will now play a sound when loading them with a syringe.

## Why It's Good For The Game
You don't have to examine them anymore to know if they are filled.

## Changelog
:cl:
imageadd: Syringe guns will now show if they are loaded or not.
soundadd: Syringe guns now play a sound when giving them a syringe.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 1]
```diff
@@ -33,6 +34,7 @@
 /obj/item/gun/syringe/process_chamber()
 	if(chambered && !chambered.BB) //we just fired
 		recharge_newshot()
+	update_icon_state()
```
Comment (ShizCalev) [inline code/modules/projectiles/guns/misc/syringe_gun.dm:37]: update_icon_state() shouldn't be called directly. use update_icon() instead.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 2]
```diff
@@ -62,11 +65,17 @@
...
+			update_icon_state()
+			playsound(src, load_sound, 40)
 			return TRUE
 		else
 			to_chat(user, "<span class='warning'>[src] cannot hold more syringes!</span>")
 	return FALSE
 
+/obj/item/gun/syringe/update_icon_state()
+	if (max_syringes == 1)
+		icon_state = "syringegun[syringes.len > 0 ? "" : "_empty"]"
```
Comment (ShizCalev) [inline code/modules/projectiles/guns/misc/syringe_gun.dm:77]: there's a bunch of subtypes below that'll have the wrong sprite after being loaded now.

also, should probably just make this an overlay.

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/projectiles/guns/misc/syringe_gun.dm (hunk 2 above)
Comment (Jared-Fogle) [reply code/modules/projectiles/guns/misc/syringe_gun.dm:77]: Forgot about the stuff other than the rapid syringe gun. What would you recommend here? A flag that all subtypes tick off, or explicitly checking that we're not in a subtype?

**Comment by ShizCalev** [comment]
oof, didn't mean to push that directly to your branch xD

**Comment by ShizCalev** [comment]
Also fixed the glass on the basic syringe gun not being transparent.

**Review by Twaticus — APPROVED** [review]

**Comment by Jared-Fogle** [comment]
@ShizCalev Well, that's one way to do it I suppose 😛 

Though it appears you only set the variable for the rapid syringe gun. There's other subtypes, like the DNA syringe gun and the blow gun. Any reason you didn't set those, or did you just forget?

**Comment by ShizCalev** [comment]
@Jared-Fogle I set the var for the blowgun since that doesn't need an overlay.

As for the others;
image

**Review by kriskog — APPROVED** [review]

**Pull request merged**